lib/authn.js

```diff
@@ -143,21 +143,20 @@ function _addAuthOptions({options, config}) {
  *
  * @private
  * @param {object} options - Options to use.
- * @param {object} options.config - Config options.
+ * @param {object} options.options - Connection options.
  *
  * @returns {Promise<object>} Returns an object with the db and admin.
  */
-async function _getUnauthenticatedDb({config}) {
+async function _getUnauthenticatedDb({options}) {
   // only use the config stuff with out auth credentials
```

Comment:
This comment is not correct any more, right? And is this the right thing to do since this is supposed to be an authenticated DB connection?